src/main/java/nl/tudelft/ti2206/group9/audio/AudioPlayer.java

```diff
@@ -69,6 +69,27 @@ public final void play() {
 	}
 
 	/**
+	 * Check whether an AudioPlayer is already running or not,
+	 * and if not, also starts the AudioPlayer.
+	 */
+	public final void playCheck() {
+		if (!this.isRunning()) {
+			this.play();
+		}
+
+	}
+
+	/**
+	 * Check whether an AudioPlayer is already running or not,
+	 * and if so, also stop the AudioPlayer.
+	 */
+	public final void stopCheck() {
+		if (this.isRunning()) {
+			this.stop();
+		}
+	}
```

Comment:
This has to do with different needs of soundtracks and sound effects for play(). Sound effects just have to be played, but soundtracks have to be checked for looping. 
I agree that play() can be refactored, so I just did, by adding a boolean.
These four methods are now refactored into two. :)